samples/graph-meeting-notification/nodejs/package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "meeting-notification-bot",
+    "version": "1.0.0",
+    "description": "Bot Builder v4 Bot Teams sample",
+    "author": "Microsoft",
+    "license": "MIT",
+    "main": "index.js",
+    "scripts": {
+        "start": "nodemon ./index.js",
+        "watch": "nodemon ./index.js",
+        "lint": "eslint .",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Microsoft/BotBuilder-Samples.git"
+    },
+    "dependencies": {
+        "@microsoft/microsoft-graph-client": "^3.0.2",
+        "adaptivecards-templating": "^2.2.0",
+        "axios": "^0.21.1",
+        "botbuilder": "^4.16.0",
+        "crypto": "^1.0.1",
+        "dotenv": "^10.0.0",
+        "express": "^4.17.1",
+        "isomorphic-fetch": "^3.0.0",
+        "jszip": "^3.6.0",
```

Comment:
Are we using all of these packages, remove if not in use like   "jszip": "^3.6.0" etc.